module/observationSubmissions/helper.js

```diff
@@ -135,6 +135,12 @@ module.exports = class ObservationSubmissionsHelper {
                 observationSubmissionsDocument['entityTypeId'] = entityTypeDocumentsAPICall.data[0]._id;
               }
 
+              if(observationSubmissionsDocument.entityInformation.externalId){
```

Comment:
@borkarsaish65 this api is not pushing data to kafka. then why we are attaching parent information here. we should only add the values in events . moke sure it not  getting stored in database